core/validation/initial_check.py

```diff
@@ -21,7 +21,7 @@ def extract_submission_details(
     :param reporting_round: Integer representing the round being ingested.
     :return: A dictionary containing the extracted submission details.
     """
-    details_dict = {"ValueChecks": {}, "NullChecks": {}}
+    wrong_input_checks = {}
```

Comment:
Could the docstring be updated to reflect this more accurately

Reply:
Updated docstring and pushed